### ISSUE TITLE
feat(tooltip): trigger tooltips with hover and focus by default

### ISSUE THREAD

**Issue by bastienmoulia** [ng-bootstrap/ng-bootstrap#3022]
Don't know if it's an oversight or if it's the right way to do it, but in Bootstrap the tooltips are triggered by hover and focus, and ng-boostrap use only the hover.

https://getbootstrap.com/docs/4.3/components/tooltips/#options
VS
https://ng-bootstrap.github.io/#/components/tooltip/api

**Comment by mendeza** [comment]
Thank you for mentioning this, Bastien. It’s needed for WCAG compliance.

_
Thanks,
Aaron

________________________________

**Comment by maxokorokov** [comment]
Indeed looks like the default value should be: `"hover focus"`

A quick test shows that `"hover focusin:focusout"` works fine. 
We don't have a `"focus"` alias in the `triggers.ts`, so it should be added there to allow for `"hover focus"` to work

P.S. popover should stay with `"click"` as default

**Comment by maxokorokov** [comment]
This should be an easy PR, but if anybody wants to work on this, please add an e2e test for focus handling

**Comment by chigaotr** [comment]
It looks pretty straightforward updating the Tooltip default to `"hover focus"` in the tooltip config, and adding `"focus"` alias to `triggers.ts`

Will submit a PR alone with e2e test as mentioned.
